samples/AppConfiguration/Generated/AppConfigurationModelFactory.cs

```diff
@@ -10,20 +10,20 @@ namespace AppConfiguration.Models
     /// <summary> Model factory for models. </summary>
     public static partial class AppConfigurationModelFactory
     {
-        /// <summary> Initializes a new instance of Key. </summary>
+        /// <summary> Initializes a new instance of <see cref="Key"/>. </summary>
         /// <param name="name"></param>
         /// <returns> A new <see cref="Models.Key"/> instance for mocking. </returns>
         public static Key Key(string name = null)
         {
-            return new Key(name);
+            return new Key(name, default);
```

Comment:
Just curious, isn't it a problem when service add a new property to this model and the constructor becomes like `internal Key(string name, string additionalProperty)`?